components/user-button.tsx

```diff
@@ -0,0 +1,28 @@
+import {
+  DropdownMenu,
+  DropdownMenuContent,
+  DropdownMenuItem,
+  DropdownMenuTrigger,
+} from "@/components/ui/dropdown-menu";
+import { Avatar, AvatarFallback, AvatarImage } from "@/components/ui/avatar";
+import { SignOutButton, useUser } from "@clerk/clerk-react";
+
+export const UserButton = () => {
+  const user = useUser();
+
+  return (
+    <DropdownMenu>
+      <DropdownMenuTrigger>
+        <Avatar>
+          <AvatarImage src={user.user?.imageUrl} alt="@shadcn" />
+          <AvatarFallback>CN</AvatarFallback>
```

Comment:
The fallback is for when there's an issue with the image, so instead of CN let's also pass in the user name here.